Error when running `stack exec courseography docs`

```
app/Response/Calendar.hs:16:16: error:
    Module ‘Config’ does not export ‘fallStartDate’
   |
16 | import Config (fallStartDate,
   |                ^^^^^^^^^^^^^

app/Response/Calendar.hs:17:16: error:
    Module ‘Config’ does not export ‘fallEndDate’
   |
17 |                fallEndDate,
   |                ^^^^^^^^^^^

app/Response/Calendar.hs:18:16: error:
    Module ‘Config’ does not export ‘winterStartDate’
   |
18 |                winterStartDate,
   |                ^^^^^^^^^^^^^^^

app/Response/Calendar.hs:19:16: error:
    Module ‘Config’ does not export ‘winterEndDate’
   |
19 |                winterEndDate,
   |                ^^^^^^^^^^^^^
courseography: callCommand: stack exec haddock -- -o doc -h --optghc=-iapp --optghc=-XOverloadedStrings --optghc=-XPartialTypeSignatures --optghc=-XScopedTypeVariables --ignore-all-exports app/Main.hs (exit 1): failed
```

The above result was from not copying the DevelopmentConfig.hs to app/Config.hs.

Now the error is as follow

```
app/Export/GetImages.hs:34:44: error:
    • No instance for (aeson-1.4.2.0:Data.Aeson.Types.FromJSON.FromJSON
                         (M.Map T.Text T.Text))
        arising from a use of ‘decode’
      There are instances for similar types:
        instance (aeson-1.4.2.0:Data.Aeson.Types.FromJSON.FromJSONKey k,
                  Ord k, aeson-1.4.2.0:Data.Aeson.Types.FromJSON.FromJSON v) =>
                 aeson-1.4.2.0:Data.Aeson.Types.FromJSON.FromJSON
                   (containers-0.5.11.0:Data.Map.Internal.Map k v)
          -- Defined in ‘aeson-1.4.2.0:Data.Aeson.Types.FromJSON’
    • In the second argument of ‘($)’, namely
        ‘decode $ fromStrict $ pack graphInfo’
      In the expression:
          fromMaybe M.empty $ decode $ fromStrict $ pack graphInfo ::
            M.Map T.Text T.Text
      In an equation for ‘graphInfoMap’:
          graphInfoMap
            = fromMaybe M.empty $ decode $ fromStrict $ pack graphInfo ::
                M.Map T.Text T.Text
   |
34 |     let graphInfoMap = fromMaybe M.empty $ decode $ fromStrict $ BC.pack graphInfo :: M.Map T.Text T.Text
   |                                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
courseography: callCommand: stack exec haddock -- -o doc -h --optghc=-iapp --optghc=-XOverloadedStrings --optghc=-XPartialTypeSignatures --optghc=-XScopedTypeVariables --ignore-all-exports app/Main.hs (exit 1): failed

```

--- a/stack.yaml
+++ b/stack.yaml
@@ -6,14 +6,8 @@ packages:
 - .
 allow-newer: true
 extra-deps:
-- binary-0.8.7.0
-- HaTeX-3.20.0.0
-- containers-0.5.11.0
-- graphviz-2999.20.0.3
-- happstack-server-7.5.1.3
-- sendfile-0.7.9
-- wl-pprint-extras-3.5.0.5
+- happstack-server-7.6.1
 compiler-check: newer-minor
-resolver: lts-13.19
+resolver: lts-14.27
 dump-logs: all
 require-stack-version: ! '>= 1.6'

